rex/renewable_resource.py

```diff
@@ -23,14 +23,21 @@ class SolarResource(BaseResource):
     --------
     resource.BaseResource : Parent class
     """
-    def get_SAM_df(self, site):
+    def get_SAM_df(self, site, extra_cols=None):
         """
         Get SAM solar resource DataFrame for given site
 
         Parameters
         ----------
         site : int
-            Site to extract SAM DataFrame for
+            Site to extract SAM DataFrame for.
+        extra_cols : dict, optional
```

Comment:
Nice, i like this. Always a good idea to add kwargs that can be endlessly expanded (instead of something like sufrace_albedo=True just to add the single new col).

Reply:
Totally agreed!